src/content/docs/bot-protection/identifying-bots.mdx

```diff
@@ -27,4 +27,18 @@ for an example. Our recommendation is to block requests without `User-Agent`
 headers because most clients always send this header.
 :::
 
+## Known bots
+
+Arcjet's list of known bots is comprised of two parts:
+
+1. The [bot list](https://arcjet.com/bot-list) shipped with the SDK provides human
+   readable identifiers for known bots.
+2. The identifiers on the bot list are generated from [a collection of known
+   bots](https://github.com/arcjet/well-known-bots) which includes details of
+   their owner and any variations.
```

Comment:
@davidmytton I suppose you might want to add a third item about regenerated Wasm so users don't think they update both places and are good-to-go.

Reply:
Good idea: https://github.com/arcjet/arcjet-docs/pull/70